### DISCUSSION TITLE
Inquiry about Setting Specific Window Size or Sequence Length in Autogluon

### DISCUSSION THREAD

**Discussion by SyedKumailHussainNaqvi** [autogluon/autogluon#3901] [Q&A]
Hello Autogluon Community,

I am currently working with Autogluon and have a question regarding the configuration of window size or sequence length in the context of my task.
I have thoroughly gone through the documentation, but I couldn't find specific information on how to set a custom window size or sequence length for my use case. I am working on PV power forecasting, and I would like to set the sequence length(window size) to 36. Additionally, my dataset has a time step of 5 minutes and and the predict length(horizon) is 9.

Could someone kindly guide me on how to configure Autogluon for this specific parameter? If there are any examples or code snippets available, that would be greatly appreciated.

Thank you in advance for your assistance!

train_data = TimeSeriesDataFrame.from_data_frame(
    df,
    id_column="id",
    timestamp_column="timestamp"
)
predictor = TimeSeriesPredictor(
    prediction_length=9,
    path="autogluon-m4-hourly",
    target="Active_Power",
      freq='5min',
    
    eval_metric="MASE",
)
predictor.fit(
    train_data,
    presets="best_quality",
    time_limit=900,
)

**Comment by shchur** [comment]
Hi, sorry for a slow response. I guess that this discussion is closely related to #3922, so we can continue the discussion there.
